Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PhysicalPressEventRouter.cs

```diff
@@ -44,6 +53,11 @@ public void OnHandPressTouched()
                     routingTarget.SetPress(false);
                 }
             }
+
+            if (uiButton != null && InteractableOnClick == PhysicalPressEventBehavior.EventOnTouch)
+            {
+                uiButton.onClick.Invoke();
```

Comment:
I like the executeEvents approach!
Though I wouldn't put this into pressablebutton right away. We would have to move that EventBehavior enum over as well and we can't get rid of the router class without taking care of interactable, which I wouldn't want to put on Adam right now. 
So just to avoid having a version where users have to set this up in different locations depending on if it's unity ui or interactable I would say we leave this in the router until I find time to fix this for Interactable as well. We're currently looking into migration scripts, so as soon as that's in place it should hopefully be an easy one to just move those fields from the router into pressablebutton

Reply:
if we add this we should adjust the doc comments and InteractableOnClick member as well so it's not only referring to interactable